File: Csharp_Source_Code/Structures/Characteristic.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CFX.Structures
{
    /// <summary>
    /// A singular characteristic that has been applied to a production unit
    /// </summary>
    public class Characteristic
    {
        /// <summary>
        /// The name of the characteristic
        /// </summary>
        public string Name
        {
            get;
            set;
        }

        /// <summary>
        /// The value of the characteristic
        /// </summary>
        public string Value
        {
            get;
            set;
        }
    }
}

File: CFX/Messages/Structures/Characteristic.py
from dataclasses import dataclass
from dataclasses_json import dataclass_json
from typing import Optional


@dataclass_json
@dataclass
class Characteristic(object):
    """
    A singular characteristic that has been applied to a production unit
    """
    
    Name: Optional[str] = None
    Value: Optional[str] = None